tools/opcode_metadata/generate_py_opcode_metadata.py

```
@@ -0,0 +1,107 @@
+    for opcode in sorted(opcodes, key=lambda op: op.cpython_name):
+        if opcode.rust_name not in specialized:
+            continue
+        out.write(f"{INDENT}'{opcode.cpython_name}': {opcode.id},\n")
+
+    out.write("}\n")
+
+    # opmap
+    out.write("\n")
+    out.write("opmap = {\n")
+
+    key = lambda op: (op.cpython_name not in PRIORITY_OPMAP, op.id)
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)